test_files/super.ts

```diff
@@ -0,0 +1,25 @@
+class SuperTestBase {
+  constructor(public x: number) {}
+}
+
+// A ctor with a parameter property.
+class SuperTestDerivedParamProps extends SuperTestBase {
+  constructor(public y: string) {
+    super(3);
+  }
+}
+
+// A ctor with an initialized property.
+class SuperTestDerivedInitializedProps extends SuperTestBase {
+  y: string = 'foo';
+  constructor() {
+    super(3);
+  }
+}
+
+// A ctor with a super() but none of the above two details.
+class SuperTestDerivedOrdinary extends SuperTestBase {
```

Comment:
This case doesn't seem too useful. I think I can argue that if the other two pass this one has to pass.